### DISCUSSION TITLE
文件上传组件和分步表单组件一起使用，上传失败还可以继续提交表单

### DISCUSSION THREAD

**Discussion by cds803** [alibaba/formily#3013] [Q&A 2.x] [answered]
复现步骤：
1、填写输入框，点击「下一步」按钮，进入「第二步」
2、进入「第二步」，选择上传文件，当文件上传失败时，此时无法进入「下一步」
3、点击「上一步」按钮，再点击「下一步」，此时文件组件校验失效，可以进入「第三步」

https://codesandbox.io/s/admiring-shannon-pir9hc?file=/App.tsx

**Comment by janryWang** [comment, marked as answer]
麻烦提个bug，后面这种可以直接复现问题的，直接提bug即可

**Reply by cds803** [reply]
bug已提 #3017
